packages/edit-site/src/components/dataviews/field-actions.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { DropdownMenu, MenuGroup, MenuItem } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { moreVertical } from '@wordpress/icons';
+
+function FieldActions( { item, actions } ) {
```

Comment:
This the menu we show in each row in the table, it has been extracted to be reused under the "featured image" in the grid view as well.